### ISSUE TITLE
Trusted Advisor broken for General Purpose SSD and Magnetic EBS volume storage

### ISSUE THREAD

**Issue by naorc12** [jantman/awslimitchecker#341]
## Bug Report

When reporting a bug in awslimitchecker, please provide all of the following information,
as well as any additional details that may be useful in reproducing or fixing
the issue:

### Version

awslimitchecker 4.0.1

### Installation Method

pip install awslimitchecker
Not virtualenv

### Supporting Software Versions

Python 2.7.6
Description:	Ubuntu 14.04.5 LTS

### Actual Output

Our script use awslimitchecker, we started get default value for "General Purpose (SSD) volume storage (GiB)" (102400) since yesterday for all of our regions.

### Expected Output
The real limit value.

### TrustedAdvisor

Business support.

### Testing Assistance

Indicate whether or not you will be able to assist in testing pre-release
code for the feature.

**Comment by jantman** [comment]
@naorc12 I'm not sure I completely understand the issue. Can you please provide some debug-level output that shows the problem, as well as a clear explanation of what the previous output was and what you're now seeing?

**Comment by naorc12** [comment]
Hi jantman thanks for the quick response!

Here is the output from Trusted advisor service:
<img width="873" alt="screen shot 2018-03-20 at 21 38 19" src="https://user-images.githubusercontent.com/31617960/37679977-36fd1b14-2c8b-11e8-8ec8-0365421e76ab.png">

And here is the output for running "awslimitchecker -r ap-northeast-1 -l -vv":
[limitcheckerDebug.log](https://github.com/jantman/awslimitchecker/files/1831034/limitcheckerDebug.log)

Please checkout EBS/General Purpose (SSD) volume storage (GiB) parameter which is the default(102400) instead 634880 as showed in Trusted Advisor.
It is happening to all of our regions and started about 24 hours ago. 

Thanks

**Comment by jantman** [comment]
@naorc12 Ugh. I want to bang my head against the wall...

The telling lines in your debug output are:

```
2018-03-20 19:49:42,719 [INFO trustedadvisor.py:415 - awslimitchecker.trustedadvisor._update_services() ] TrustedAdvisor returned check results for unknown limit 'General Purpose SSD (gp2) volume storage (GiB)' (service EBS)
2018-03-20 19:49:42,719 [INFO trustedadvisor.py:415 - awslimitchecker.trustedadvisor._update_services() ] TrustedAdvisor returned check results for unknown limit 'Magnetic (standard) volume storage (GiB)' (service EBS)
```

I'm now seeing this in my accounts as well.

It appears that in the last 24 hours, Trusted Advisor has **renamed two of their check results**:

"General Purpose (SSD) volume storage (GiB)" to "General Purpose SSD (gp2) volume storage (GiB)"
"Magnetic volume storage (GiB)" to "Magnetic (standard) volume storage (GiB)"

I'm going to wait 1-2 days to see if they revert this change or not, and open a ticket on my Enterprise support contract (though it probably won't do any good) as this has now broken us as well.

If this is still happening in 1-2 days, I'll cut a patch release to switch to using the new check names from Trusted Advisor.

**Comment by jantman** [comment]
I received confirmation from AWS that this change is permanent, and was done to unify the formatting of check names. They recommended that we use check IDs rather than names, but that doesn't really work for this program.

I'll try to get a patch release out today or tomorrow.

**Comment by jantman** [comment]
This has been released in 4.0.2 and is [now live on PyPI](https://pypi.python.org/pypi/awslimitchecker/4.0.2).

**Comment by naorc12** [comment]
Thanks, it's works!
